docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx

```diff
@@ -195,58 +187,24 @@ This will output something like this:
 12.49840000 ICP
 ```
 
-Once you have your ICP tokens ready, you will need to create a cycles wallet if you haven't already. This will require your identity principal, which you can obtain by running the command:
-
-```bash
-dfx identity get-principal
-```
-
-Then, to create a cycles wallet, run the following command:
-
-```bash
-dfx ledger --network ic create-canister <PRINCIPAL_ID> --amount <ICP_TOKENS>
-```
-
-Replace the **PRINCIPAL_ID** with your identity's principal value that was returned from the `dfx identity get-principal` command, then replace **ICP_TOKENS** with the amount of tokens you'd like to convert into cycles.
+Once you have your ICP tokens ready, you will need to convert them into cycles. To convert ICP into cycles using the cycles ledger, use the command:
 
-For example, if your principal is `tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe` and you want to convert 3.5 ICP tokens into cycles, the command would be:
-
-```bash
-dfx ledger --network ic create-canister tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe --amount 3.5
 ```
-
-This command will process for a few moments, and will output something that resembles:
-
-```bash
-Transfer sent at BlockHeight: 351220
-Canister created with id: "gastn-uqaaa-aaaae-aaafq-cai"
+dfx cycles convert AMOUNT --network ic
 ```
 
-The ID in this output is the address of the canister where your wallet will live. In this example, it would be `gastn-uqaaa-aaaae-aaafq-cai`.
-
-Now that your cycles wallet canister has been created, you need to install the wallet code with the command:
-
-```bash
-dfx identity --network ic deploy-wallet <CANISTER_ID>
-```
-
-Replace the **CANISTER_ID** with the value you received in the output of the previous command, such as:
-
-```bash
-dfx identity --network ic deploy-wallet gastn-uqaaa-aaaae-aaafq-cai
-```
+:::info
+This workflow utilizes the **cycles ledger** feature. If you'd like to use the **cycles wallet** instead, [view the cycles wallet documentation](/docs/current/developer-docs/getting-started/cycles/cycles-wallet).
 
-The output of this command will resemble the following:
+Please note that the cycles wallet will be deprecated in the future.
```

Comment:
I think the language around this should be more along the lines of 
`Please note that the cycles wallet will be removed from dfx in a future release.`

We aren't _deprecating_ the cycles wallet - we will still continue to provide maintenance, and folks will be able to use it / deploy it themselves. We will however, encourage folks to use the Cycles Ledger unless they know what they're doing.